### DISCUSSION TITLE
String variable doesn't rendered

### DISCUSSION THREAD

**Discussion by hitriyvalenok** [mobxjs/mobx#2938] [Q&A] [answered]
Hello! I have a variable I wanna be reactive. Example is simple: user types in an email's input, value reactively renders on the page. But it doesn't happens :( Variable's value changed, but doesn't rendered. Example on [codesandbox](https://codesandbox.io/s/mobx-reactive-var-doesnt-updates-ulp6m). Thank you!

**Comment by BATCOH** [comment, marked as answer]
In MobX v6 you should call makeObservable / makeAutoObservable, even with decorators.
See also:
https://mobx.js.org/enabling-decorators.html
https://mobx.js.org/migrating-from-4-or-5.html

**Reply by hitriyvalenok** [reply]
Thank you! Solved.
